src/PythonMigrationViewExtension/Controls/VisualDifferenceViewer.xaml

```diff
@@ -0,0 +1,74 @@
+﻿<Window x:Class="Dynamo.PythonMigration.Controls.VisualDifferenceViewer"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:ui="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
+        xmlns:local="clr-namespace:Dynamo.PythonMigration.Controls"
+        xmlns:diffplex="clr-namespace:DiffPlex.Wpf.Controls;assembly=DiffPlex.Wpf"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800"
+        Title="Migration Assistant"
+        WindowStartupLocation="CenterOwner"
+        Background="#272822">
+
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="44" />
+        </Grid.RowDefinitions>
+
+        <diffplex:DiffViewer x:Name="DiffView"
+                             OldTextHeader="Old"
+                             NewTextHeader="New"
+                             FontFamily="Consolas"
+                             FontSize="10pt"
+                             Background="#272822"
```

Comment:
can some of these colours come from the Dynamo styles ? Not sure if they exist, but if they do, we should try and use them  (unless it requires adding a dependency on another project)

Reply:
taking these from the `ScriptEditorWindow`, but might make sense it to the `DynamoModernDictionary` and get them from there in both places

Reply:
if they are in the dictionary as usable WPF styles, then we should try - unless it means the additional dependency